src/resources/views/list.blade.php

```diff
@@ -231,120 +236,24 @@
-        $('#crudTable tbody td .details-row-button').parent('td')
-          .removeClass('details-control').addClass('details-control')
-          .removeClass('text-center').addClass('text-center')
-          .removeClass('cursor-pointer').addClass('cursor-pointer');
-
-        // Add event listener for opening and closing details
-        $('#crudTable tbody td.details-control').on('click', function () {
-            var tr = $(this).closest('tr');
-            var btn = $(this).find('.details-row-button');
-            var row = table.row( tr );
-
-            if ( row.child.isShown() ) {
-                // This row is already open - close it
-                btn.removeClass('fa-minus-square-o').addClass('fa-plus-square-o');
-                $('div.table_row_slider', row.child()).slideUp( function () {
-                    row.child.hide();
-                    tr.removeClass('shown');
-                } );
-            }
-            else {
-                // Open this row
-                btn.removeClass('fa-plus-square-o').addClass('fa-minus-square-o');
-                // Get the details with ajax
-                $.ajax({
-                  url: '{{ url($crud->route) }}/'+btn.data('entry-id')+'/details',
-                  type: 'GET',
-                  // dataType: 'default: Intelligent Guess (Other values: xml, json, script, or html)',
-                  // data: {param1: 'value1'},
-                })
-                .done(function(data) {
-                  // console.log("-- success getting table extra details row with AJAX");
-                  row.child("<div class='table_row_slider'>" + data + "</div>", 'no-padding').show();
-                  tr.addClass('shown');
-                  $('div.table_row_slider', row.child()).slideDown();
-                  register_delete_button_action();
-                })
-                .fail(function(data) {
-                  // console.log("-- error getting table extra details row with AJAX");
-                  row.child("<div class='table_row_slider'>{{ trans('backpack::crud.details_row_loading_error') }}</div>").show();
-                  tr.addClass('shown');
-                  $('div.table_row_slider', row.child()).slideDown();
-                })
-                .always(function(data) {
-                  // console.log("-- complete getting table extra details row with AJAX");
-                });
-            }
-        } );
-      }
-
-      register_details_row_button_action();
-      @endif
-
+      // when columns are hidden by reponsive plugin,
+      // the table should have the has-hidden-columns class
+      crud.table.on( 'responsive-resize', function ( e, datatable, columns ) {
+          if (crud.table.responsive.hasHidden()) {
+            $("#crudTable").removeClass('has-hidden-columns').addClass('has-hidden-columns');
+           } else {
+            $("#crudTable").removeClass('has-hidden-columns');
+           }
+      } );
 
 	  });
```

Comment:
Looks something is wrong with indentation. I'd suggest to keep changes code style changes minimal for this PR and if we want to sort out the indentation issues we could do it in one commit afterwards.

Reply:
Agreed! Thank you for checking this out @Sobak !